web/main.ts

```diff
@@ -1110,6 +1337,19 @@ class GitGraphView {
 						}, 'Pushing Branch');
 					}, target);
 				}
+			}, {
+				title: 'Pull Branch' + ELLIPSIS,
+				visible: visibility.pull && this.gitRemotes.length > 0,
+				onClick: () => {
+					dialog.showForm('Are you sure you want to update the local branch <b><i>' + escapeHtml(refName) + '</i></b> with the latest changes from <b><i>' + escapeHtml(this.gitRemotes[0] + '/' + refName) + '</i></b>?', [{
+						type: DialogInputType.Checkbox,
+						name: 'Force Update',
+						value: this.config.dialogDefaults.fetchIntoLocalBranch.forceFetch,
+						info: 'Force the local branch to be reset to the remote branch (discard local commits).'
+					}], 'Yes, update', (values) => {
+						runAction({ command: 'fetchIntoLocalBranch', repo: this.currentRepo, remote: this.gitRemotes[0], remoteBranch: refName, localBranch: refName, force: <boolean>values[0] }, 'Updating Branch');

```

Comment:
This statement:

`remote: this.gitRemotes[0]`
 
In my case the array contains:

gitRemotes: [    "bendera",    "origin",    "qrakee",    "upstream"]

but my local branch is only on origin. So it fails.